src/components/Dropdown/dropdown.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '@/styles/mediaQueries.scss';
+@import '@/styles/variables.scss';
+@import '@/styles/colors.scss';
+
+.dropdown {
+  position: relative;
+  /* display: inline-block; */
+
+  .dropdownBtn {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    background-color: $white;
+    border: 1.5px solid $black;
+    border-radius: $brad-sm;
+    padding: 0.5rem;
+    padding-top: 0.75rem;
+
+    .arrowBtn {
+      width: 1.3rem;
+      height: 1.3rem;
+
+      &:hover {
+        cursor: pointer;
+      }
+    }
+  }
+
+  .radiusNull {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+
+  .dropdownContent {
+    position: absolute;
+    width: 100%;
+    height: 40rem;
```

Comment:
Height is quite large for mobile screens.
Would go for a combination of min-height and max-height here, for example:
```css
min-height: 25rem;
max-height: 40rem;
```
Exact values to be determined.

Reply:
in the inspector, 40rem is what you need to cover the BTNs, 45rem on the bigger screens. alternative: 39rem and 44rem if the lower tip of the BTNs should be visible (maybe it looks better). 
didn't use min/max so that they're really covered, otherwise confusing for user and there's a need to create a modal with an outside click to close it.

Reply:
i adjusted it to 'min-height:39rem' and 'max-height: 40rem', if you think min/max is better, but it seems a bit odd to me

Reply:
Yes, you're right, this makes sense.